terraform/alerting/main.tf

```diff
@@ -0,0 +1,10 @@
+module "verification-server" {
```

Comment:
Can we import a module into another module?

Reply:
Running `tf apply` on my test project seem to work as intended.